source/Nevermore/Util/DataModificationQueryBuilder.cs

```diff
@@ -75,17 +76,22 @@ public PreparedCommand PrepareUpdate(object document, UpdateOptions options = nu
                 default:
                     throw new ArgumentOutOfRangeException();
             }
-            
+
+            var whereStatements = new List<string> {$"[{mapping.IdColumn.ColumnName}] = @{mapping.IdColumn.ColumnName}"};
+            whereStatements.AddRange(mapping.RowVersionColumns().Select(dvc => $"[{dvc.ColumnName}] = @{dvc.ColumnName}"));
```

Comment:
Can you have more than 1 column of type `RowVersion`?

Reply:
I've seen examples where more than one column was used but we can be opinionated here.